markdown/preface.markdown

```diff
@@ -136,37 +136,30 @@ is a powerful package for plotting in two dimensions (and basic 3D). It draws it
 http://matplotlib.org/
 * **IPython**
 is an interactive interface for Python, which allows you to quickly interact with your data and test ideas.
+https://ipython.org/
 * The **Jupyter**
 notebook runs in your browser and allows you to write code in line with text and mathematical expressions, displaying the results of computation within the text.
 This entire book has been written with Jupyter.
 Jupyter started out as an IPython extension, but now supports multiple languages, including Cython, Julia, R, Octave, Bash, Perl and Ruby.
 http://jupyter.org
 * **pandas**
-provides fast, columnar data structures in an easy-to-use package, particularly to work with labelled data sets such as tables or relational databases, and manage time series.
-It also has some handy data analysis tools such as for data parsing and cleaning, sliding windows, aggregation and plotting.
+provides fast, columnar data structures in an easy-to-use package.
+It is particularly suited to working with labelled data sets such as tables or relational databases, and for managing time series data.
+Pandas also has some handy data analysis tools for data parsing and cleaning, aggregation, analysis using sliding windows, and plotting.
```

Comment:
"Data analysis tools for data parsing" is awkward, more so with "data analysis tools for data [...] analysis using sliding windows". So this sentence needs a bit more work.